aiocache/decorators.py

```diff
@@ -55,7 +55,9 @@ def cached_decorator(func):
 
             try:
                 if await cache_instance.exists(cache_key):
-                    return await cache_instance.get(cache_key)
+                    result = await cache_instance.get(cache_key)
+                    await cache_instance.close()
```

Comment:
I agree with you on reusing the connection for the decorators, this is something I have in mind because there are a couple of edge cases.

Regardless of reusing or not the connection, its not worth it to delay the response to the user waiting for the close to pool. I prefer having a message saying Task X wasn't executed rather than unneeded delayed responses. We can think how to close those pending connections later :)